template/en/default/bug/create/create.html.tmpl

```diff
@@ -63,43 +63,17 @@ function initCrashSignatureField() {
                       'cf_crash_signature_action', 'cf_crash_signature');
   [% END %]
 }
-
-var initialowners = new Array([% product.components.size %]);
+var data = [% data %];
```

Comment:
Even though we think data is json, it's important to make the browser parse it so arbitrary JS can never leak in.
var data = JSON.parse('[% data FILTER js %]');